base/redactor.go

```diff
@@ -137,6 +137,8 @@ func (l *RedactionLevel) MarshalText() ([]byte, error) {
 // UnmarshalText unmarshals text to a RedactionLevel.
 func (l *RedactionLevel) UnmarshalText(text []byte) error {
 	switch strings.ToLower(string(text)) {
+	case "unset":
+		*l = RedactionLevelDefault
```

Comment:
This should be `RedactUnset` really, in the same way that MarshalText returns "unset".

Want to keep the "default" handling fairly isolated and keep the JSON/Text marshaling accurate to what is being passed in/out.